vnext/Desktop/WebSocket.cpp

```diff
@@ -493,142 +597,188 @@ void SecureWebSocket::Handshake(const IWebSocket::Options &options) {
 
 #pragma endregion IWebSocket static members
 
-namespace Test {
-
+namespace Test
+{
 #pragma region MockStream
```

Comment:
Can we move this into its own file?

Reply:
I'm afraid not.
Note, those classes ultimately don't make it to the Desktop DLL, but because of the heavily template-based nature of Boost/Beast, defining those types in a different compilation unit would yield linking errors.

At the same time, I don't want to make these implementations completely header-based, to improve binary artifact reusability and decrease build time.